hub/powertoys/hosts-file-editor.md

```diff
@@ -0,0 +1,41 @@
+---
+title: PowerToys Hosts File Editor utility for Windows
+description: Hosts File Editor is a convenient way to edit hosts file.
+ms.date: 10/31/2022
+ms.topic: article
+no-loc: [PowerToys, Windows, Hosts File Editor, Win]
+---
+
+# Hosts File Editor utility
+
+Hosts File Editor provides a convenient way to edit hosts file configuration.
```

Comment:
Should we explain a bit more context about what the hosts file configuration is and how it can be used?

Something like.. "Windows includes a local 'Hosts' file that contains domain names and matching IP addresses, acting as a map to identify and locate hosts on IP networks. Every time you visit a website, your computer will check the hosts file first to see which IP address it connects to. If the information is not there, your internet service provider will look into the DNS for the resources to load the site.

The Hosts File Editor provides a convenient way to edit the hosts file configuration. 

This can be useful for scenarios like migrating a website to a new hosting provider or domain name, which may take a 24-48 hour period of downtime. Creating a custom IP address to associate with your domain using the hosts file can enable you to see how it will look on the new server."

Not sure about that scenario bit... just trying to come up with some context that may help folks unfamiliar with the host file concept.

Reply:
Thank you, have added an explanation as you suggested.